scripts/qiita

```diff
@@ -59,9 +62,24 @@ def pet():
     pass
 
 
-@pet.group()
-def webserver():
-    pass
+@pet.group(invoke_without_command=True)
```

Comment:
Should this instead be part of the 'start' command? IMO it's a bit weird that we are actually calling a group rather than the command.